From 5e07232f2f285cbb86111184aab24470bfbdace8 Mon Sep 17 00:00:00 2001
From: Jorge Olmos <jorge.olmos@flexvdi.com>
Date: Wed, 12 Apr 2017 09:18:34 +0200
Subject: [PATCH] Add m4 path for autoreconf

When cross-compiling (or compiling to a given prefix), m4 macros
are not installed in a system path.
Latest autoreconf does not honor ACLOCAL_FLAGS environment variable,
so we need to tell it where are the macros.
The missing macro is AM_GLIB_GNU_GETTEXT installed by glib.
---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index f29b02b..8a14b8f 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,4 +1,4 @@
-ACLOCAL_AMFLAGS = -I m4
+ACLOCAL_AMFLAGS = -I m4 ${ACLOCAL_FLAGS}
 NULL =
 
 SUBDIRS = spice-common src man po doc data
-- 
2.7.4

